src/actix/api/query_api.rs

```diff
@@ -80,11 +83,21 @@ async fn query_points(
             .map(api::rest::ScoredPoint::from)
             .collect_vec();
 
-        Ok(QueryResponse { points })
+        Ok((QueryResponse { points }, usage))
```

Comment:
we also want to return usage even in a case when operation failed on later stages.